recipes/keycloack-gatekeeper/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+go_compiler:
+  - go-nocgo
```

Comment:
You can remove this file since the `go_compiler` variant is part of the conda_build_config.yaml from conda-forge.